sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/README.md

```diff
@@ -1,49 +1,45 @@
-### Configure necessary API permissions
-The sample retrieves user's group memberships using Microsoft graph API which requires the registered app to have `Directory.AccessAsUser.All` permission under `Delegated Permissions` of `Microsoft Graph`, which can allow an application to access the directory as the signed-in user. Also, to display the function of calling multiple resources, this sample will acquire `ActivityFeed.Read` permission under `Office 365 Management APIs` resource. You need AAD admin privilege to be able to grant the permission in API ACCESS -> Required permission. You can follow the below steps:	
-
-* In the list of pages for the app, select **API permissions**	
-   - Click the **Add a permission** button	
-   - Ensure that the **Microsoft APIs** tab is selected	
-   - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**	
-   - In the **Delegated permissions** section, ensure that the right permissions are checked: **Directory.AccessAsUser.All**	
-   - Select the **Add permissions** button
-   - Under **Office 365 Management APIs** tab
-   - Select **Delegated permissions**, and then click **ActivityFeed.Read**
-   - Select the **Add permissions** button
-   - Click **Grant Permissions...** and Yes when prompted.
+### Configure web app
+1. Search for and select your tenant in **Azure Active Directory**.
+2. Under Manage In the same tenant, select **App registrations** -> **New registration**.![Protal manage](docs/image-protal-manage.png "Protal manage")
+3. The registered application name is filled into *webapp*, select **Accounts in this organizational directory only**, click the **register** button.![Register a web app](docs/image-register-a-web-app.png "Register a web app")
+4. Under **webapp** application, select **Certificates & secrets** -> **new client secret**, expires select **Never**, click the **add** button.(Remember to save the secrets here and use them later)![Creat secrets](docs/image-creat-secrets-app.png "Creat secrets")
+5. Under **webapp** application, select **Authentication** -> **Add a platform**, select **web** platform, redirect urls set to *http://localhost:8080/login/oauth2/code/azure*, check the Access Token and ID Token checkboxes, click **configure** button.![Add a platfform](docs/image-add-a-platfform.png "Add a platfform")
+6. Under **webapp** application, select **Authentication** -> **Add URI**, you need to add redirect URIs of *http://localhost:8080/login/oauth2/code/arm*. ![App add url](docs/image-app-add-url.png "App add url")
+7. Under **webapp** application, select **API permissions** -> **Add a permission**, select **Microsoft Graph**. Next, search **Calendar.Read** via select Permissions, check the check box, click **add permissions** button.![Api permission](docs/image-api-permissions.png "Api permission")
```

Comment:
No `Calendar.Read`. There is no problem to add it, but to keep the scopes same to integration test, it's better delete `Calendar.Read`.